person.rb

```diff
@@ -22,6 +23,11 @@ def can_use_services?
     of_age?
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] Your add_rental method should look something like below to make a has_many association 👍
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```